src/site/stages/build/process-cms-exports/transformers/paragraph-button.js

```diff
@@ -5,9 +5,15 @@ const { getDrupalValue } = require('./helpers');
 const transform = entity => ({
   entityType: 'paragraph',
   entityBundle: 'button',
+  fieldButtonLabel: getDrupalValue(entity.fieldButtonLabel),
+  fieldButtonLink: {
+    url: {
+      path: entity.fieldButtonLink[0].uri,
+    },
+  },
```

Comment:
IDK why the content export representing the button has `uri` whereas the GraphQL version has `path`